### ISSUE TITLE
[SF][Card][A11y]: Unnecessary tab focus appeared in the non-interactive card header

### ISSUE THREAD

**Issue by ericchan1983** [SAP/ui5-webcomponents#3642]
**Describe the bug**
The tab focused on the card header even though the interactive is false

**Isolated Example**
https://sap.github.io/ui5-webcomponents-react/?path=/docs/data-display-card--default-story

**To Reproduce**
![image](https://user-images.githubusercontent.com/1614528/129168960-99870095-ebcd-40cb-9c78-8aac885a6eb9.png)

**Expected behavior**
The non-interactive card header shouldn't have tab focus.

**Screenshots**

**UI5 Web Components for React Information**
`@ui5/webcomponents` version:  0.31.14
`@ui5/webcomponents-react` version: 0.131.13
Operating System:  Mac OS
Browser: Chrome

**Additional context**

**Comment by GerganaKremenska** [comment]
Hello @ericchan1983,

After further consultation with our ACC experts, the behavior of having the focus on the card header is always correct. Even if the property headerInteractive is false.

This is recommended for two reasons:

The ARIA standard allows for inactive elements to be focusable in some contexts. Especially inside of composite widgets. In the case of the card header it is actually recommended, because if the header is not focusable, it will hide its presence from the screen reader users. If we do not focus the header aways we will loose some screen reader announcements. This is described in more details in this part of the ARIA documentation.
A common usage of the cards is to have several of them on a single page - like a dashboard. If you have a mix of cards which are interactive and cards which are not interactive next to each other, we still want to be consistent in where the focus lands. This will make it easier for people with disabilities to navigate.
If you have further questions please reopen the issue.

Best Regards,
Gergana Kremenska
